sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -130,6 +162,14 @@ export function ndJsonPolicy(): PipelinePolicy;
 // @public
 export const ndJsonPolicyName = "ndJsonPolicy";
 
+// @public
+export type ParsedWWWAuthenticate = {
+    [Key in ValidParsedWWWAuthenticateProperties]?: string;
+};
+
+// @public
+export function parseWWWAuthenticate(wwwAuthenticate: string): ParsedWWWAuthenticate;
```

Comment:
why is this method in `core-rest-pipeline`? Seems like a great candidate for `core-util`

Reply:
Do we want to take a dependency on core-util? currently the only dependency is tslib and I like to keep the list minimal.  Maybe we can move this policy to core-client? I feel CAE is a bit higher level than REST.

Reply:
Move www-authenticate parsing to core-xml.  the default on-challenge should be removed since it doesn't do anything now.